lib/matplotlib/tests/test_font_manager.py

```diff
@@ -31,6 +31,20 @@ def test_font_priority():
     assert cmap[8729] == 30
 
 
+def test_ttflist_weight():
+    assert all(isinstance(f.weight, six.string_types)
+               for f in fontManager.ttflist)
+
+
+def test_score_weight():
+    assert (0 ==
+            fontManager.score_weight("regular", "regular") ==
+            fontManager.score_weight("bold", "bold") <
+            fontManager.score_weight("normal", "regular") ==
+            fontManager.score_weight(400, 400) <
```

Comment:
Why is 400/400 (which are equal) at the same level as normal/regular (which are not equal)? Is it because they're numbers and not strings?